Library/Formula/gnu-apl.rb

```diff
@@ -1,9 +1,9 @@
 class GnuApl < Formula
+  version "1.5"
   desc "GNU implementation of the programming language APL"
   homepage "https://www.gnu.org/software/apl/"
-  url "http://ftpmirror.gnu.org/apl/apl-1.4.tar.gz"
-  mirror "https://ftp.gnu.org/gnu/apl/apl-1.4.tar.gz"
-  sha256 "69da31db180e6ae17116122758c266e19f62e256f04e3c4959f6f2b224a1893a"
+  url "svn://svn.sv.gnu.org/apl/trunk", :revision => "662"
```

Comment:
I tried. The tagged version 1.5 won't work.

http://svn.savannah.gnu.org/viewvc/tags/apl-1.5/?root=apl